roles/nvidia-gpu-operator/tasks/k8s.yml

```diff
@@ -11,10 +11,6 @@
   command: /usr/local/bin/helm repo update
   changed_when: false
 
-- name: Remove existing GPU Operator labels to accommodate potenial configuration changes
-  command: kubectl label node --all nvidia.com/gpu.deploy.container-toolkit-  nvidia.com/gpu.deploy.dcgm- nvidia.com/gpu.deploy.dcgm-exporter- nvidia.com/gpu.deploy.device-plugin- nvidia.com/gpu.deploy.driver- nvidia.com/gpu.deploy.gpu-feature-discovery- nvidia.com/gpu.deploy.node-status-exporter- nvidia.com/gpu.deploy.operator-validator-
```

Comment:
I had originally added this line to work around what seemed like a bug in the GPU Operator, but in retrospect this line should not be necessary as all these values are dynamically configured/set by the GPU Operator at install time.

Removing this because it is not needed and the reliance on kubectl in a role was causing failures.